portfolio/src/main/webapp/style.css

```diff
@@ -33,6 +33,48 @@
     opacity: 0.5;
 }
 
+.planet-imgs img {
+    max-height: 300px;
+    max-width: 300px;
+}
+
+.planet-imgs img:hover {
+    opacity: 0.5;
+}
+
+.vegan {
+    position: absolute;
+    left: 110px;
+    top: 300px;
+    opacity: 0;
+}
+
+.clothes {
+    position: absolute;
+    left: 350px;
+    top: 300px;
+    opacity: 0;
+}
+
+.plastic {
+    position: absolute;
+    left: 600px;
+    top: 300px;
+    opacity: 0px;
```

Comment:
Don't need px here

Reply:
Thank you for the comment! I changed that later on, it should be in a later pull request. Should I still change it here?

Reply:
Sorry, you can ignore my above comment - I changed it!